pattern-hues.lic

```diff
@@ -0,0 +1,77 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#pattern_hues
+=end
+
+custom_require.call(%w(common))
+no_pause_all
+
+class PatternHues
+  include DRC
+
+  def initialize
+    arg_definitions = [
+    [
+      { name: 'timing', regex: /now/i, optional: true, description: 'Run now or continual.' },
+    ]
+    ]
+
+    args = parse_args(arg_definitions)
+    @settings = get_settings
+    now = Time.now
+    duration = @settings.pattern_hues['duration']
+
+    adj = @settings.pattern_hues['pattern_hues_styles']['pattern'] ||
+      adj = ['bright', 'crystalline', 'gleaming', 'glimmering', 'glistening',\
```

Comment:
You don't need the second assignment (`adj =`) (same with line 29)

Reply:
Removed. I am testing another item and will push once I feel it is working as I expect.